tests/availability_adx.codegen.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { test, expect } from '@playwright/test';
+
```

Comment:
Are these codegen classes generated as part of a npm lifecycle ? say something similar to npm generate or something similar ? if so they should not be checked in.